### PR TITLE
Updating devkit-spriter

### PR DESCRIPTION
**Pull request by Javasamurai**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "archiver": "^0.14.4",
     "bluebird": "^2.9.34",
     "chalk": "^1.1.1",
-    "devkit-spriter": "0.0.9",
+    "devkit-spriter": "git+https://github.com/hashcube/devkit-spriter.git#master-fbinstant",
```
Comment (rampr) [inline package.json:14]: why did we not just retain the es6 branch ?

Comment (Javasamurai) [reply package.json:14]: That can be done.But I though it's better to make a separate branch for fbinstant itself.

**Review by rampr — APPROVED** [review]

**Comment by rampr** [comment]
The title is misleading though. We just added additional functionality on top of the existing spriter.
Nothing was broken and fixed per se.

**Review by vivekb91 — APPROVED** [review]

**Comment by Javasamurai** [comment]
@rampr Updated the title too.

**Pull request merged**